### PR TITLE
Raise error if content quality service does not return 200 response code

### PR DESCRIPTION
**Pull request by KyleMacPherson**
Use GovukError.notify to to notify Sentry of the error.

### CODE REVIEW

**Comment by KyleMacPherson** [comment]
[(2) Fix: Undefined method `dig' for String](https://trello.com/c/u9AWrC3a/397-2-fix-undefined-method-dig-for-string)

File: app/etl/item/content_quality_service.rb [hunk 1]
```diff
@@ -42,4 +43,6 @@ def convert_results(response, content)
   def count_metric(response, metric_name)
     response.dig(metric_name, 'count') || 0
   end
+
+  class NoContentError < StandardError; end
```
Comment (pmanrubia) [inline app/etl/item/content_quality_service.rb]: Not sure...  If response code is not `200` we don't know what the error is. I would not named it `NoContentError`. I'd rather rename it to `QualityMetricsError` (or similar), and I would add the error to the exception so we can track it. What do you think?

File: app/etl/items/jobs/import_quality_metrics_job.rb [hunk 2]
```diff
@@ -3,5 +3,13 @@ class Items::Jobs::ImportQualityMetricsJob < Items::Jobs::ApplicationJob
 
   def run(*args)
     Items::Importers::QualityMetrics.run(*args)
+  rescue StandardError => e
+    GovukError.notify(
```
Comment (pmanrubia) [inline app/etl/items/jobs/import_quality_metrics_job.rb]: Could you please use one line for notifying the error?

File: spec/etl/items/jobs/import_quality_metrics_job_spec.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+RSpec.describe Items::Jobs::ImportQualityMetricsJob do
```
Comment (pmanrubia) [inline spec/etl/items/jobs/import_quality_metrics_job_spec.rb]: 👍

File: app/etl/item/content_quality_service.rb [hunk 4]
```diff
@@ -16,6 +16,7 @@ def fetch(content)
       body: { content: content }.to_json,
       headers: { 'Content-Type' => 'application/json' }
     )
+    raise QualityMetricsError("response body: #{response.body}") unless response.code == 200
```
Comment (KyleMacPherson) [inline app/etl/item/content_quality_service.rb]: @pmanrubia wasn't sure exactly what you meant. Something like this?

**Pull request closed without merging**